src/elm/clinical.coffee

```diff
@@ -116,7 +116,7 @@ module.exports.CalculateAge = class CalculateAge extends Expression
 
   exec: (ctx) ->
     date1 = @execArgs(ctx)
-    date2 = dt.DateTime.fromDate(new Date())
+    date2 = dt.DateTime.fromDate(ctx.getExecutionDateTime())
```

Comment:
Right now, `ctx.getExecutionDateTime()` can return `null`, in which case I believe `fromDate` would enter its `else` block and throw an error.

Reply:
So, it was clear from the beginning that we needed to make sure there is always an `executionDateTime` even if its the default one. As the engine is currently built, there will always be a valid 'executionDateTime' if `PatientContext` or `PopulationContext` are used, because all other contexts branch from the base `Patient` and `Population` contexts. So if there is a case where `ctx.getExecutionDateTime()` returns null, then errors should occur. Where and what those errors should be / look like we can discuss in the morning.

Reply:
@cmoesel any thoughts on this?

Reply:
Perhaps the `getExecutionDateTime` function should throw an error if one is not set?  Does that sound reasonable?

Reply:
That works for me. I will make this change now.